src/java/picard/analysis/directed/CollectHsMetrics.java

```diff
@@ -45,15 +45,50 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Collects a set of Hybrid Selection specific metrics from an aligned SAM" +
-                "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
-                "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
-                "mean coverage information for every target.",
-        usageShort = "Collects Hybrid Selection-specific metrics for a SAM or BAM file",
+        usage = CollectHsMetrics.USAGE_SUMMARY + CollectHsMetrics.USAGE_DETAILS,
+        usageShort = CollectHsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
+
 )
 public class CollectHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
 
+    static final String USAGE_SUMMARY = "Collects hybrid-selection (HS) specific metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "These metrics enable users to determine the efficacy of HS experiments.  " +
+            "In addition, if a reference sequence is provided, this program will calculate AT/GC dropout metrics and enable users to invoke the" +
+            " PER_TARGET_COVERAGE option to output both GC and mean coverage information for every target. <br /><br />" +
+            "" +
+            "Both CollectTargetedPCRMetrics and CalculateHybridSelection metrics share virtually identical program structures except " +
+            "for the name of their targeting mechanisms (e.g. bait set or amplicon set).  Both depend on the TargetMetricsCollector " +
+            "and collect metrics on all reads in the INPUT SAM/BAM file. <br /><br /> " +
+            "" +
+            "Hybrid selection (HS) enables targeted sequencing analysis via the capture of specified genomic DNA " +
```

Comment:
Too much detail about hybrid selection. This should be replaced by a link to the GATK dictionary.